tempodb/encoding/vparquet3/block_findtracebyid.go

```diff
@@ -125,10 +125,10 @@ func (b *backendBlock) FindTraceByID(ctx context.Context, traceID common.ID, opt
 		span.SetTag("inspectedBytes", rr.BytesRead())
 	}()
 
-	return findTraceByID(derivedCtx, traceID, b.meta, pf, rowGroup)
+	return findTraceByID(derivedCtx, traceID, opts.MaxBytes, b.meta, pf, rowGroup)
 }
 
-func findTraceByID(ctx context.Context, traceID common.ID, meta *backend.BlockMeta, pf *parquet.File, rowGroup int) (*tempopb.Trace, error) {
+func findTraceByID(ctx context.Context, traceID common.ID, maxTraceSizeBytes int, meta *backend.BlockMeta, pf *parquet.File, rowGroup int) (*tempopb.Trace, error) {
```

Comment:
Default encoding still is `vParquet2`. We should add it there as well.

Reply:
I did not realize that. We should switch default to 3